### ISSUE TITLE
frequent Updating error code in warning log

### ISSUE THREAD

**Issue by charlesdagenais** [claudegel/sinope-130#90]
is this expected behavior to show this warning so frequently in the log if no error code ? looks like it's logged on pretty much each update

```
Logger: custom_components.neviweb130.climate
Source: custom_components/neviweb130/climate.py:902
Integration: Sinope Neviweb130 (documentation)
First occurred: 7 août 2022 à 22:18:12 (434 occurrences)
Last logged: 09:53:20

Updating error code: {'referenceSensor': 'ok', 'compensationSensor': 'ok', 'airSensor': 'ok', 'wireSensor': 'ok', 'currentOverload': 'ok', 'thermalOverload': 'ok', 'loadError': 'ok', 'endOfLife': False}
```

[line 902](https://github.com/claudegel/sinope-130/blob/3ed4195c169421d9292c4e455e298afe1b78ca5d/custom_components/neviweb130/climate.py)

**Comment by claudegel** [comment]
Hi, this logging come from when I've added support for error message from climate devices on Neviweb. You can comment out the line 902 safely by adding a «#» in front of it. I'll remove it on next release.
Thank you for letting me know about it.

**Comment by claudegel** [comment]
I've published version 1.4.1 to clean unnecessary logging. Thank you. Let me know if you find other logging that are useless
